kernel/kselftest.py

```diff
@@ -92,10 +90,21 @@ def test(self):
         """
         Execute the kernel selftest
         """
+        count = False
```

Comment:
This is not a counter anymore, that's why I suggested the name of this variable to be `errors`.